packages/manager/modules/web-universe-components/src/order/model/pricing.class.js

```diff
@@ -0,0 +1,24 @@
+import get from 'lodash/get';
+
+/** Class represeting a catalog Pricing.
+ *  For class properties, see the order.catalog.public.Pricing[]
+ *  in /order/catalog/public{/productName}
+ */
+export default class Pricing {
+  /**
+   * constructor
+   *
+   * @param {Object} options  Class properties grouped.
```

Comment:
Maybe mention that we are supposed to pass a `order.catalog.public.Catalog` parameter here?
Or Am I wrong?

Reply:
https://github.com/ovh/manager/pull/2072/files/8188b5fa3f3d6d631669d2bf4d4b3edd35993226#diff-c295ed16e2f1e2cd1d97ae0a70e74c75R4 ;)